src/client/clientenvironment.cpp

```diff
@@ -393,33 +376,26 @@ u16 ClientEnvironment::addActiveObject(ClientActiveObject *object)
 	return object->getId();
 }
 
-void ClientEnvironment::addActiveObject(u16 id, u8 type,
-	const std::string &init_data)
+void ClientEnvironment::addActiveObject(u16 id, u8 type, const std::string &init_data)
 {
-	ClientActiveObject* obj =
-		ClientActiveObject::create((ActiveObjectType) type, m_client, this);
-	if(obj == NULL)
-	{
-		infostream<<"ClientEnvironment::addActiveObject(): "
-			<<"id="<<id<<" type="<<type<<": Couldn't create object"
-			<<std::endl;
+	ClientActiveObject *obj =
+			ClientActiveObject::create((ActiveObjectType)type, m_client, this);
```

Comment:
I personally think `(ActiveObjectType) type` is more readable than `(ActiveObjectType)type`.

Reply:
same

Reply:
Totally agreed.